openfisca_us/parameters/gov/states/il/tax/income/personal_exemption/senior_and_blind_exemption.yaml

```
@@ -0,0 +1,12 @@
+description: Illinois personal exemption allowance
+values:
+  2021-01-01:
```

Don't need `value` unless providing metadata for a particular value
```suggestion
  2021-01-01: 1_000
```